src/index.tsx

```diff
@@ -312,7 +318,11 @@ export class App extends React.Component<{}, AppState> {
         onCopy={e => {
           e.clipboardData.setData(
             "text/plain",
-            JSON.stringify(elements.filter(element => element.isSelected))
+            JSON.stringify(
+              elements
+                .filter(element => element.isSelected)
+                .map(({ shape, ...el }) => el)
```

Comment:
This is a smart way to copy without!

Reply:
I removed the shape element from history, clipboard, and exported JSON file. Is there any other feature in the app that needs these exports to be removed.